### PR TITLE
Make the sysfile that is read by the composer model manager dynamic

### PR DESCRIPTION
**Pull request by mbwhite**
System model is present but that is then used at build time to create a fixed class that is used by the model manager. 

This is needed as the runtime can't load files from disk. Being a different file makes it easier to develop, however.

### CODE REVIEW

**Comment by sstone1** [comment]
@mbwhite if we're generating files in the build, can we make sure they're added to `.gitignore` please?

**Comment by mbwhite** [comment]
@sstone1 it is ;-)

**Comment by sstone1** [comment]
@mbwhite oh yes - so why can I see it in the list of new files then?

File: packages/composer-common/scripts/systemModelgen.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const targetFile = path.resolve(__dirname, '..', 'lib','systemModel.js');
+const templatePath = path.resolve(__dirname, '..', 'lib','_template');
+const modelFile = path.resolve(__dirname, '..', 'lib','system.cto');
+
+let env = nunjucks.configure(templatePath,{autoescape:false});
+
+// get the template file
+let model = fs.readFileSync(modelFile,  { encoding: 'utf8' });
+// let template = fs.readFileSync(templateFile,  { encoding: 'utf8' });
```
Comment (jt-nti) [inline packages/composer-common/scripts/systemModelgen.js]: Commented code

**Pull request merged**